packwerk.gemspec

```diff
@@ -50,6 +50,7 @@ Gem::Specification.new do |spec|
   # For Ruby parsing
   spec.add_dependency("ast")
   spec.add_dependency("parser")
+  spec.add_dependency("prism", ">= 0.23.0") # 0.24.0 fixes a performance issue with the parser translator
```

Comment:
Let's change this to 0.24.0

Reply:
I was goign to but somehow I only changed the comment 🤦🏼